day2/part2.py

```diff
@@ -0,0 +1,26 @@
+import re
+import copy
+
+def multiplyList(myList):
+    result = 1
+    for x in myList:
+        result = result * int(x)
+    return result
+
+
+f = open("data.txt", "r")
+balls = {
+    'red': 0,
+    'green': 0,
+    'blue': 0
+}
+total_count = 0
+for line in f:
+    line_list = re.split('; |, |: ', line.rstrip('\n '))
+    max_no_balls = copy.deepcopy(balls)
```

Comment:
You probably do not need a deepcopy here. 😄 
You can just ** the dict into the dict function but 👍 for the style points. xd